sphinx/source/docs/user_guide/embed.rst

```diff
@@ -60,7 +60,7 @@ can be used in HTML documents however you like:
     plot = figure()
     plot.circle([1,2], [3,4])
 
-    script, div = components(plot)
```

Comment:
I think leave this in? It's correct for the particular case of passing in a single plot object and the subsequent text about the returned script is confusing without it I think